src/test/java/com/wikia/webdriver/testcases/verbatimtests/VerbatimTests.java

```diff
@@ -0,0 +1,27 @@
+package com.wikia.webdriver.testcases.verbatimtests;
+
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.verbatim.widgets.TwitterPageObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership: Content X-Wing
+ */
+public class VerbatimTests extends NewTestTemplate {
```

Comment:
verbatim is just a name of tag in mediawiki. The tested functionality is a widget. 
change the class name to widgetTests or thirdparties